doc/source/train/common/torch-configure-run.rst

```diff
@@ -12,6 +12,14 @@ Outside of your training function, create a :class:`~ray.train.ScalingConfig` ob
     scaling_config = ScalingConfig(num_workers=2, use_gpu=True)
 
 
+3. :class:`resources_per_worker <ray.train.ScalingConfig>` - Used to specify other resource requirements, such as non-GPU accelerators like HPUs.
+
+.. testcode::
+
+    from ray.train import ScalingConfig
+    scaling_config = ScalingConfig(num_workers=2, resources_per_worker={"HPU": 1})
+
+
```

Comment:
We can move this code into `using-gpus.rst` as an example of how to use other accelerators, i.e., HPU with ray train. And use that file as a single source of truth of applying non-gpu accelerators. But for the general torch example, we may not want to add extra code examples here.